src/serve/minibf/mod.rs

```diff
@@ -16,7 +16,11 @@ pub struct Config {
 
 pub async fn serve(
     cfg: Config,
+    genesis: Arc<Genesis>,
+    genesis_values: GenesisValues,
```

Comment:
GenesisValues from traverse is deprecated. We shouldn't need this. Why is it used for?

Reply:
This ones: https://github.com/txpipe/pallas/blob/c08d766d38604d13786863332a99edb99455189b/pallas-traverse/src/time.rs#L121-L144